docs/tempo/website/release-notes/v1-5.md

```diff
@@ -0,0 +1,110 @@
+---
+title: V1.5
+weight: 70
+---
+
+# Version 1.5 release notes
+
+The Tempo team is pleased to announce the release of Tempo 1.5.
+
+With this release, Tempo adds Parquet block support (experimental), anonymous usage reporting to help the Tempo team and Grafana understand how Tempo is used, and enhancements to the metrics-generator. 
+
+We encourage users to upgrade to Grafana 9.1 to see additional Tempo-powered capabilities,
+such as the [application performance management (APM) dashboard](https://grafana.com/docs/tempo/latest/metrics-generator/app-performance-mgmt/) which visualizes span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+
+>**Note**: For a complete list of changes and enhancements, refer to the [Tempo 1.5 changelog](https://github.com/grafana/tempo/releases). 
+
+
+## Features and enhancements
+
+Version 1.5 contains the following features and enhancements. 
+
+### Apache Parquet block format support
+
+Tempo now has a columnar block format based on Apache Parquet.
+A columnar block format may result in improved search performance and also enables a large ecosystem of tools access to the underlying trace data.
+
+This feature is experimental.
+
+While Parquet can be used as a drop-in replacement, Parquet requires more CPU and memory resources.
```

Comment:
```suggestion
The new Parquet block format can be used as a drop-in replacement for Tempo's existing block format. No data conversion or upgrade process is necessary. As soon as the Parquet format is enabled, Tempo starts writing data in that format, leaving existing data as-is. 
```

Reply:
I assume that both types of data are queryable? Like you suddenly don't lose the ability to query your traces in the old non-parquet format? Maybe we should state that explicitly?

Reply:
That's my understanding.